src/app/page.tsx

```diff
@@ -1,11 +1,13 @@
-"use client";
+import HomePage from "@/components/HomePage";
 import styles from "./page.module.css";
-import { MapComponent } from "@/components/Map";
+import { getMapData } from "@lib/loadMap";
+import { useMemo } from "react";
 
 export default function Home() {
+  const mapData = useMemo(() => getMapData(), []);
```

Comment:
I’m actually a bit surprised that this is working, because `getMapData()` is an _async_ function! So, in theory, you’d need to `await` the result and also build some kind of loading/waiting logic here (instead of just using `useMemo()`). But I guess for a first prototype this is fine. :D